src/Docker.DotNet/Endpoints/ISwarmOperations.cs

```diff
@@ -63,7 +63,7 @@ public interface ISwarmOperations
         /// 503 - Node not part of a swarm.
         /// </remarks>
         /// <param name="parameters">The leave parameters.</param>
-        Task LeaveSwarmAsync(SwarmLeaveParameters parameters = null, CancellationToken cancellationToken = default(CancellationToken));
+        Task<DockerApiResponse> LeaveSwarmAsync(SwarmLeaveParameters parameters = null, CancellationToken cancellationToken = default(CancellationToken));
```

Comment:
Aren't these all breaking changes? @galvesribeiro - Do we care anymore of is it worth making this a major release since its the first large update in a long time.

Reply:
I used this internally to retrieve Docker response, restored to avoid breaking changes.